examples/scripts/dpo_online.py

```diff
@@ -69,40 +82,40 @@
     model = AutoModelForCausalLM.from_pretrained(
         model_config.model_name_or_path, trust_remote_code=model_config.trust_remote_code, **model_kwargs
     )
-    ref_model = AutoModelForCausalLM.from_pretrained(
-        model_config.model_name_or_path, trust_remote_code=model_config.trust_remote_code, **model_kwargs
-    )
+
     reward_model = AutoModelForSequenceClassification.from_pretrained(
-        training_args.reward_model_path, num_labels=1, trust_remote_code=model_config.trust_remote_code
+        training_args.reward_model_path,
```

Comment:
If I'm not mistaken, one needs to specify the number of labels to 1 because the default will be 2. 

```suggestion
        training_args.reward_model_path,
        num_labels=1,
```

Reply:
This is strange that it runs without it...